src/app/components/filebrowsermvs/filebrowsermvs.component.ts

```diff
@@ -155,8 +177,20 @@ export class FileBrowserMVSComponent implements OnInit, OnDestroy {//IFileBrowse
 
   onNodeRightClick(event:any) {
     let node = event.node;
-    // TODO: Add right click properties menu to Datasets via Editor/Explorer interaction    
-    this.log.debug(`Node right click at ${event.clientX},${event.clientY}, off=${event.offsetX},${event.offsetY}, node=`,node);
+    let rightClickProperties = this.rightClickPropertiesDataset;
+
+    if (this.windowActions) {
+      let didContextMenuSpawn = this.windowActions.spawnContextMenu(event.originalEvent.clientX, event.originalEvent.clientY, rightClickProperties, true);
+      // TODO: Fix Zowe's context menu such that if it doesn't have enough space to spawn, it moves itself accordingly to spawn.
+      if (!didContextMenuSpawn) { // If context menu failed to spawn...
```

Comment:
While we do need to fix this, this is not a guaranteed fix: what if it fails twice? May want to loop?

Reply:
This may not be ideal because context menu can fail for various reasons, and a loop would just crash the app unless we put a counter. But if it doesn't show after the offset (the node height) it probably isn't showing for a different reason